tests/everflow/conftest.py

```diff
@@ -0,0 +1,26 @@
+import pytest
+import logging
+
+
+@pytest.fixture(autouse=True, scope="package")
+def setup_recycle_port(duthosts, tbinfo):
```

Comment:
Not seeing this function used elsewhere.

Reply:
The autouse being set makes it run automatically at the start of the first everflow test and cleanup at the very end.  It sets up the IP configuration on the recycle ports which we need for everflow to forward out the tunnel.